app/routes/ui/scenario.py

```diff
@@ -50,13 +50,22 @@ async def get(self, scenario_friendly_name, scene_number=0):
                 )
 
             is_last_scene = scene_number_int == len(scenario_obj.scene_ids) - 1
+            background_ext = (
+                "." + scene_dict["background_details"]["s3_key"].split(".")[-1]
```

Comment:
Does [-1] mean the last element in the array? Or a typo?

Reply:
Yeah last element in array!

Reply:
Yeah basically trying to get the extension of the file, so would be splitting by "." and getting last element